raiden/tests/scenarios/ci/long_path_mediated_transfers.yaml

```diff
@@ -1,365 +1,391 @@
+            - assert: {from: 0, to: 1, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 1, to: 0, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 1, to: 2, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 2, to: 1, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 2, to: 3, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 3, to: 2, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 3, to: 4, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 4, to: 3, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 4, to: 5, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 5, to: 4, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 5, to: 6, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 6, to: 5, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 6, to: 7, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 7, to: 6, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 7, to: 8, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 8, to: 7, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 8, to: 9, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 9, to: 8, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 9, to: 10, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 10, to: 9, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 10, to: 11, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 11, to: 10, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 11, to: 12, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 12, to: 11, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 12, to: 13, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 13, to: 12, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 13, to: 14, total_deposit: 1_000, balance: 1_000, state: "opened"}
+            - assert: {from: 14, to: 13, total_deposit: 1_000, balance: 1_000, state: "opened"}
       - serial:
-          name: "Make one transfer from 0 to 1"
+          name: "Make one transfer from node0 to node1"
           tasks:
-            - transfer: {from: 0, to: 1, amount: 1, expected_http_status: 200}
+            - transfer: {from: 0, to: 1, amount: 1, expected_http_status: 200, lock_timeout: 30}
```

Comment:
Please also use higher amounts for the payments themselves like in the other scenarios